test/framework/resources/templates/logger-app.yaml.tpl

```diff
@@ -18,7 +18,7 @@ spec:
     spec:
       containers:
       - name: logger
-        image: registry.k8s.io/e2e-test-images/agnhost:2.40
+        image: eu.gcr.io/gardener-project/3rd/agnhost:2.40
```

Comment:
Do we need to document this as guideline somewhere? Otherwise I could image someone introduces similar image to be used in Shoot and the image signing part breaks again. Now how we are supposed to update this image? Do we need to do some image signing magic. It would be nice if such things are documented somewhere.

Reply:
Tests that deploy images in the shoot are not affected, only the logging test as it deploys workload in the seed cluster where image signing validation might be enabled. 

As now this is the only image affected by the image signing and there are multiple other tests already using images from eu.gcr.io/gardener-project/3rd, I decided to choose the quick option to do it manually. 

Long term, this is not sustainable of course and I have some ideas in my mind how this to be overcome, but will need some more time to get better understanding of how the test machinery works and what can be done where.

Reply:
TL;DR
1. If we want to update the version of `eu.gcr.io/gardener-project/3rd/agnhost` what we need to do? I guess we have to image somehow, right?
2. I suggest to add docs to time being how to add image that is used in existing Seed cluster and how to update such existing image as this is not obvious to me.

Reply:
> If we want to update the version of `eu.gcr.io/gardener-project/3rd/agnhost` what we need to do? I guess we have to sign the image somehow, right?

To update the image, it is enough just to replicate it, for example 
```bash
$ crane copy registry.k8s.io/e2e-test-images/agnhost:2.40 eu.gcr.io/gardener-project/3rd/agnhost:2.40
```

With regards to the signing part, I cannot share anything publicly unfortunately. Even if the image is updated and not signed, the tests will fail only in environment where signature verification is enabled. 



> I suggest to add docs for time being how to add image that is used in existing Seed cluster and how to update such existing image as this is not obvious to me.

I am afraid I do not completely understand this one. It is this test https://github.com/gardener/gardener/blob/406ccb88d4980b6247fd8cad3a7519ada5db1201/test/testmachinery/shoots/logging/shoot_logging.go#L141 that deploys workload into the seed.

Reply:
@vpnachev was kind to raise a PR describing how such an image can be signed.